hathor/event/websocket/protocol.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2022 Hathor Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
```

Comment:
Oh, also, this shouldn't be needed at all.

Reply:
Oh, I know why you might have added this. For postponed evaluation, right? If that's the case, please solve it using the following pattern instead:

```python
if TYPE_CHECKING:
    from hathor.event.websocket import EventWebsocketFactory

# don't use this:
foo: EventWebsocketFactory 
# use this instead:
foo: 'EventWebsocketFactory'
```

Reply:
> Oh, I know why you might have added this. For postponed evaluation, right?

Yep. I have changed it to your suggestion.